software/20.Serverless/README.md

```diff
@@ -0,0 +1,100 @@
+# Workshop Serverless ✨
+Learn the basics of Serverless, an architecture helping you saving time and money.
+##### ✔️ Serverless App using [Netlify](https://www.netlify.com/), [React](https://fr.reactjs.org/) TS and [Chakra UI](https://chakra-ui.com/)
+##### ✔️ Serverless API using [Serverless Framework](https://www.serverless.com/)
+
+## Introduction 🔰
+- You want to stay focus on your code, forget about setting-up severs ?  
+- You want to have a scalable application without doing anything ?  
+- You want to pay for what you use ?  
+- You don't even know what is serverless ?  
+
+#### Well, you are in the right place at the right time 🍀
```

Comment:
Add an explanation, you shouldn't ask questions but instead, answers it.
Check the GRPC and Kafka workshop for examples